src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -979,6 +979,13 @@ pub struct Hypervisor {
     #[serde(default, flatten)]
     pub shared_fs: SharedFsInfo,
 
+    /// A sandbox annotation used to specify prefetch_files.list host path container image
+    /// being used, and runtime will pass it to Hypervisor to  search for corresponding
+    /// prefetch list file:
+    ///   prefetch_list_path = /path/to/<uid>/xyz.com/fedora:36/prefetch_file.list
+    #[serde(default)]
+    pub prefetch_list_path: String,
```

Comment:
Can we configure this path via `configuration.toml` ?

Reply:
Thanks.
It will be updated into configuration, but generally it will not be directly written in the configuration because the prefetch_files path stored for each pod is different.
So, we can only do this through annotation to specify the prefetch files path.